Sources/Core/EthereumNetwork/Request/APIRequest+ComputedProperties.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  APIRequest+ComputedProperties.swift
+//  
+//
+//  Created by Yaroslav Yashin on 12.07.2022.
+//
+
+import Foundation
+
+extension APIRequest {
+    var method: REST {
+        switch self {
+        default: return .POST
```

Comment:
Actually yes, since none of Eth API uses any other REST method. But i'd either think that it's redundant, and could be just dropped in a favor of hardcoding this method in URLSession configuration.